README.md

```diff
@@ -10,6 +10,51 @@ Our goal is to make it easy to create real-time healthcare analytics application
 
 Follow the [**Getting Started Guide**](https://developer.ibm.com/streamsdev/2017/06/07/getting-started-streams-healthcare-analytics-platform/) to learn about how to leverage the services from the Streams Healthcare Analytics Platform
 
+### Streams QSE ###
+This is necessary if you are running on a system not supported by Streams.
+1. Install [Streams QSE](http://ibmstreams.github.io/streamsx.documentation/docs/4.3/qse-intro/).
+
+### Endpoint-monitor ###
+Original repository: https://github.com/IBMStreams/endpoint-monitor  
+Updated repository: https://github.com/kenguo573/endpoint-monitor  
```

Comment:
Hi Ken,

Can you create a PR  to merge your changes to  the endpoint-monitor repository so that these instructions can point to the main repository instead?